extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -853,109 +932,110 @@ protected function  isCorporateWiki() {
-			throw new Exception("This should not be called outside of self-request context.");
+		$config = $this->getVal( 'config', false );
+		if ( !$config instanceof Wikia\Search\Config ) {
+			throw new Exception( "This should not be called outside of self-request context." );
 		}
 
-		$searchEngine = (new SearchEngine);
+		$searchEngine = ( new SearchEngine );
 
 		$searchableNamespaces = $searchEngine->searchableNamespaces();
 
-		wfRunHooks( 'AdvancedBoxSearchableNamespaces', array( &$searchableNamespaces ) );
+		wfRunHooks( 'AdvancedBoxSearchableNamespaces', [ &$searchableNamespaces ] );
 
-		$this->setVal( 'namespaces', 			$config->getNamespaces() );
-		$this->setVal( 'searchableNamespaces', 	$searchableNamespaces );
+		$this->setVal( 'namespaces', $config->getNamespaces() );
+		$this->setVal( 'searchableNamespaces', $searchableNamespaces );
 	}
 
 	/**
 	 * This is how we generate the search type tabs in the left-hand rail
+	 *
 	 * @see    SearchControllerTest::tabs
 	 * @throws Exception
 	 */
 	public function tabs() {
-		$config = $this->getVal('config', false);
+		$config = $this->getVal( 'config', false );
 
-		if ( ! $config || (! $config instanceOf Wikia\Search\Config ) ) {
-			throw new Exception("This should not be called outside of self-request context.");
+		if ( !$config || ( !$config instanceOf Wikia\Search\Config ) ) {
+			throw new Exception( "This should not be called outside of self-request context." );
 		}
 
 		$filters = $config->getFilterQueries();
 		$rank = $config->getRank();
 		$is_video_wiki = $this->wg->CityId == Wikia\Search\QueryService\Select\Dismax\Video::VIDEO_WIKI_ID;
 
-		$form = array(
-				'by_category' =>        $this->getVal('by_category', false),
-				'cat_videogames' =>     isset( $filters['cat_videogames'] ),
-				'cat_entertainment' =>  isset( $filters['cat_entertainment'] ),
-				'cat_lifestyle' =>      isset( $filters['cat_lifestyle'] ),
-				'is_hd' =>              isset( $filters['is_hd'] ),
-				'is_image' =>           isset( $filters['is_image'] ),
-				'is_video' =>           isset( $filters['is_video'] ),
-				'sort_default' =>       $rank == 'default',
-				'sort_longest' =>       $rank == 'longest',
-				'sort_newest' =>        $rank == 'newest',
-				'no_filter' =>          !( isset( $filters['is_image'] ) || isset( $filters['is_video'] ) ),
-			);
+		$form = [
+			'by_category' => $this->getVal( 'by_category', false ),
+			'cat_videogames' => isset( $filters['cat_videogames'] ),
+			'cat_entertainment' => isset( $filters['cat_entertainment'] ),
+			'cat_lifestyle' => isset( $filters['cat_lifestyle'] ),
+			'is_hd' => isset( $filters['is_hd'] ),
+			'is_image' => isset( $filters['is_image'] ),
+			'is_video' => isset( $filters['is_video'] ),
+			'sort_default' => $rank == 'default',
+			'sort_longest' => $rank == 'longest',
+			'sort_newest' => $rank == 'newest',
+			'no_filter' => !( isset( $filters['is_image'] ) || isset( $filters['is_video'] ) ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "no_filter" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)